Doc/library/enum.rst

```diff
@@ -998,13 +998,18 @@ Finer Points
 Supported ``__dunder__`` names
 """"""""""""""""""""""""""""""
 
-:attr:`__members__` is an :class:`OrderedDict` of ``member_name``:``member``
+:attr:`__members__` is a read-only ordered dictionary ``member_name``:``member``
```

Comment:
"`of`" should be after "`dictionary`".  My apologies for not noticing that the first time.

Reply:
Wouldn't be better to use "mapping" instead of "dictionary"? Technically the result is not a `dict` subclass and therefore is not fully compatible with `dict` (some APIs require only `dict` or subclasses).

Reply:
Good point.  Review updated.